.pre-commit-config.yaml

```diff
@@ -0,0 +1,23 @@
+repos:
+-   repo: git@github.com:pre-commit/pre-commit-hooks
+    sha: v1.1.0
+    hooks:
+    - id: debug-statements
+    - id: trailing-whitespace
+    - id: check-merge-conflict
+    - id: check-executables-have-shebangs
+    - id: check-ast
+    - id: check-byte-order-marker
+    - id: check-json
+    - id: check-symlinks
+    - id: check-vcs-permalinks
+    - id: check-xml
+    - id: check-yaml
+    - id: detect-aws-credentials
+      args: ['--allow-missing-credentials']
+    - id: detect-private-key
+    - id: forbid-new-submodules
+    - id: no-commit-to-branch
+      args: ['-b master']
+    - id: flake8
+args: ['--exclude=docs/*,*migrations*', '--ignore=E501']
```

Comment:
this args are for all commands? 🤔

Reply:
No. I've fixed that in other PR. Let me put the fix in this branch.